package.json

```diff
@@ -46,6 +46,7 @@
     "@types/mocha": "^9.0.0",
     "@types/node": "^16.9.4",
     "@types/node-emoji": "^1.8.1",
+    "@types/tmp": "^0.2.3",
```

Comment:
is this packaged used in this PR?

Reply:
don't need, i think because of node version mismatch it was complaining but when using it with `nvm use` it is not required